src/components/Searchbar/InstantSearch.tsx

```diff
@@ -21,37 +27,62 @@ export const InstantSearch = () => {
     setSearch(e.target.value);
 
     await axios
-        .get(`https://jsonplaceholder.typicode.com/albums`)
-        .then((res) => {
-          setPosts(res.data);
-        })
-        .catch((error) => {
-          if (axios.isCancel(error) || error) {
-            console.log('Could not get');
-          }
-        });
+      .get(`https://jsonplaceholder.typicode.com/albums`)
+      .then((res) => {
+        setPosts(res.data);
+      })
+      .catch((error) => {
+        if (axios.isCancel(error) || error) {
+          console.log('Could not get');
+        }
+      });
   };
 
   return (
     <div className="searchModule">
```

Comment:
rename the whole component InstantSearch is bad name and name of the classes too

Reply:
SearchBar.tsx or Search.tsx ??

Reply:
Just search will be okay